run_ortmodule_mvp_mnist_deepspeed.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
how is this script to be leveraged? Is it mostly for dev work? Or is it to be incorporated in ci testing somehow?

Reply:
I think we should check it in for now. Once you have the test script ready to run with CI, we should lock this in with a test point.

Reply:
To serve as a sample, especially for the new comers. It can be deleted once CI is up and running